sunpy/map/mapbase.py

```diff
@@ -2778,7 +2778,7 @@ def contour(self, level, **kwargs):
         contours = [self.wcs.array_index_to_world(c[:, 0], c[:, 1]) for c in contours]
         return contours
 
-    def get_contours(self, level, method='contourpy', **kwargs):
+    def get_contours(self, level, method='contourpy', area=False, **kwargs):
```

Comment:
I am not really pro adding another keyword but I don't have a better idea.

Hopefully the other maintainers will better ideas.